Setup/UpgradeSchema.php

```diff
@@ -220,6 +220,20 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             }
         }
 
+        if (version_compare($context->getVersion(), '2.9.4', '<')) {
+            $installer->getConnection()->changeColumn(
```

Comment:
why didn't you change InstallSchema.php?

Reply:
Because it's an upgrade.

Reply:
so what will happen for customers that install from scratch? - they will have float type and customers that upgrade will have decimal (10,2) type?

Reply:
It'll run the install first & then the upgrade. But if you'll only upgrade, it'll only run the upgrade.

Reply:
i prefer for the good order that you will change also install.

Reply:
Changed on 4.0.0 as well.